distributions/Dist_sdl433.py

```diff
@@ -6,16 +6,43 @@ def __init__(self):
 		self.f_max = 1
 		self.x_min = -1
 		self.x_max = 1
+        self.f = np.array([])
+        self.domain = np.array([])
 
-
+    #Exponential distribution starting at x in [0,1]
+    #Takes in x as an array
 	def pdf(self, x):
-		"""This is your PDF"""
-		return np.abs(x)
+
+        #Define the domain
+        x_min = 0
+        x_max = 1
+        domain = np.linspace(x_min, x_max, num = 10000)
+
+        #Define exponential function
+        f = 10 * np.e ** (-10 * x)
+
+        #Calculate distribution function at x
+        f_of_x = f[x * 10000]
+
+        #Integrate
+        Z = np.trapz(f, x, dx = 1/10000)
```

Comment:
it's nice to see you are familiar with this feature in Numpy, but I was hoping you would do it with a pencil and paper.